examples/file-sharing.rs

```diff
@@ -196,6 +208,8 @@ enum CliArgument {
         name: String,
     },
     Get {
+        #[clap(long)]
+        path: PathBuf,
```

Comment:
Instead of providing a file path for the retrieved file to be written to, how about leveraging unix pipelines?

E.g. one would do:

``` bash
cargo run -- get [...] > my_file
```

What do you think?

Reply:
Good idea! Thanks for your advices. I've changed the output to stdout().